torch/distributed/rpc.py

```diff
@@ -5,7 +5,7 @@
 from . import ProcessGroupAgent
 from . import WorkerId
 from .internal_rpc_utils import serialize, PythonUDF
-from .rpc_backend_handler import is_backend_registered, registered_init_rpc
+from .rpc_backend_registry import is_rpc_backend_registered, init_rpc_backend
```

Comment:
If I'm not mistaken, these functions will now be available on the `torch.distributed.rpc` module.

Can you add an `__all__` to make sure only the functions we want are available on the module?

Reply:
@xush6528 Can you ack this comment?

Reply:
@pietern 

Sorry.

Will it be better if we don't use `from "something" import *`?

Reply:
Thanks.

I agree, but that puts the burden on the user. I think it's good practice to define the interface and define what should be exported and what not. This is not done everywhere consistently, but still there are plenty hits for both `__all__` and `import *` in the entire code base.

Reply:
I think rpc should create a folder like `distributed/autograd` does.
Maintain the names want to expose to users in `torch/distributed/autograd/__init__.py`.
Do the actual implmentation in `torch/distributed/rpc/rpc_api.py`.